File: Angular/app/Evenements/Models/EvenementsSearchParameters.ts
export class EvenementsSearchParameters {

    constructor() {
        this.Id = null;

        this.Libelle = null;

        this.DtMin = null;
        this.DtMax = null;

        this.EleveId = null;
        this.EvenementParentId = null;
        this.TypologieId = null;

    }

    public Id: number | null;

    public Libelle: string | null;

    public DtMin: string | null;
    public DtMax: string | null;

    public EleveId: number | null;
    public EvenementParentId: number | null;
    public TypologieId: number | null;


    public OnlyParentsYN: string | null;
}

File: WS/Models/IN/EvenementsSearchParameters.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WS.Models.IN
{
    public class EvenementsSearchParameters
    {

        public Int32? Id { set; get; }
        public String Libelle { set; get; }


        public String DtMin { set; get; }
        public String DtMax { set; get; }


        public Int32? EvenementParentId { set; get; }

        public Int32? TypologieId { set; get; }

        public Int32? EleveId { set; get; }


        public String OnlyParentsYN { set; get; }

    }

}